plugin.program.dnsleaktest/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.program.dnsleaktest"
+version="1.1"
+name="DNS Leak-Test"
+provider-name="Space2Walker">
+<requires>
+  <import addon="xbmc.python" version="2.26.0"/>
+  <import addon="script.module.certifi" version="2017.07.27.1"/>
+  <import addon="script.module.chardet" version="3.0.4"/>
+  <import addon="script.module.idna" version="2.6"/>
+  <import addon="script.module.requests" version="2.19.1"/>
+  <import addon="script.module.urllib3" version="1.22"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+  <provides>executable</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="en">DNS Leak-Test Plugin for Kodi</summary>
+  <description lang="en_GB">This addon performs a DNS leak test on https://bash.ws/dnsleak!
+  The Basic Code comes from https://github.com/macvk/dnsleaktest.
+  </description>
+  <disclaimer lang="en_GB">Use add your own risk!!</disclaimer>
+  <platform>all</platform>
+  <license>GPL-3.0</license>
+  <website>https://bash.ws</website>
```

Comment:
This should be your website, not the site from bash.ws, unless it is actually Bash.ws that is creating this add-on. You could add your github page here.

Reply:
Website should point to the site which provides the service, so this is correct.

Reply:
Looking at other add-ons and their website, you seem to be right.